src/a-runtime-console/kubernetes/ui/pipeline/list-page/list-page.pipeline.spec.ts

```diff
@@ -1,63 +1,36 @@
 /* tslint:disable:no-unused-variable */
+import { NO_ERRORS_SCHEMA } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { FormsModule } from '@angular/forms';
-import { BaseRequestOptions, Http, RequestOptions } from '@angular/http';
-import { MockBackend } from '@angular/http/testing';
-import { RouterTestingModule } from '@angular/router/testing';
-import { MomentModule } from 'angular2-moment';
-import { StackDetailsModule } from 'fabric8-stack-analysis-ui';
-import { RestangularModule } from 'ngx-restangular';
-import { ModalModule } from 'ngx-modal';
-import { Fabric8CommonModule } from '../../../../common/common.module';
-import { KubernetesComponentsModule } from '../../../components/components.module';
-import { KubernetesStoreModule } from '../../../kubernetes.store.module';
-import { BuildConfigDialogsModule } from '../../buildconfig/delete-dialog/buildconfig.dialogs.module';
-import { BuildStageViewComponent } from '../build-stage-view/build-stage-view.component';
-import { InputActionDialog } from '../input-action-dialog/input-action-dialog.component';
-import { PipelinesListToolbarComponent } from '../list-toolbar/list-toolbar.pipeline.component';
-import { PipelinesListComponent } from '../list/list.pipeline.component';
-import { TestAppModule } from './../../../../app.test.module';
-import { StageTimePipe } from './../build-stage-view/stage-time.pipe';
 import { PipelinesListPage } from './list-page.pipeline.component';
 
+import { APIsStore } from '../../../store/apis.store';
+import { BuildStore } from '../../../store/build.store';
+import { BuildConfigStore } from '../../../store/buildconfig.store';
+
+import { Observable } from 'rxjs';
+
 describe('PipelinesListPage', () => {
   let component: PipelinesListPage;
   let fixture: ComponentFixture<PipelinesListPage>;
 
+
   beforeEach(async(() => {
+    let mockBuildConfigStore: any = jasmine.createSpy('BuildConfigService');
```

Comment:
@jiekang I also tried mocking the methods like this but it doesn't seem to work either.